### PR TITLE
added user db schema

### PR DESCRIPTION
**Pull request by pmoieni**
- added user db schema

## Description

Dependent #1

## Current status

- [x] Waiting for review
- [ ] Waiting for comment resolution
- [x] Waiting for merge
- [ ] Draft
- [ ] Trivial PR (nominal cosmetic/typo/whitespace changes)

## Semantic Versioning

- This is a `feat` change

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

### CODE REVIEW

**Review by adoublef — CHANGES_REQUESTED** [review]
I think before we can push to main, we need to confirm what we are storing in our DB(s), what DB we are using and the shape/schema. Once we have that sorted I think we also need to make sure we have test files included before pushing to main.

**Review by hansvb — APPROVED** [review]
Looks good except for
- the indentation in some `.yaml`-files but that is apparently not a problem?
- the fact that there are 2 topics in this 1 PR (but that's ok for now, let's try to avoid it in the future)
- the fact that there are no tests yet (which IMO is also ok for now, as we're still in very early stages)

File: req/types.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package req
+
+const (
+	GET    = "GET"
+	POST   = "POST"
+	PUT    = "PUT"
```
Comment (harveysanders) [inline req/types.go:6]: Why not use the method constants from stdlib?
https://pkg.go.dev/net/http#pkg-constants

File: cmd/server/main.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+)
+
+func main() {
+	log.Println("loading config")
+	err := rmx.LoadConfig()
+	if err != nil {
+		log.Fatalf("fatal error config file: %v", err.Error())
+	}
+
+	// database connection shouldn't be implemented here
```
Comment (harveysanders) [inline cmd/server/main.go:17]: Go ahead and delete this commented-out code if you're going to use `config.go` to setup the DB connection

File: req/types.go (hunk 1 above)
Comment (pmoieni) [reply req/types.go:6]: Didn't know about them. Will update. Thanks.

**Pull request closed without merging**